be/test/storage/tablet_updates_test.cpp

```diff
@@ -1213,7 +1213,11 @@ void TabletUpdatesTest::test_compaction_with_empty_rowset(bool enable_persistent
         keys5.push_back(i * 3 + 2);
     }
     ASSERT_TRUE(_tablet->rowset_commit(5, create_rowset(_tablet, keys5)).ok());
-    std::this_thread::sleep_for(std::chrono::milliseconds(200));
+    {
+        // used for wait merely
```

Comment:
Could the compaction wait for the completion?

Reply:
in real world procedure, the internal do wait for the completion properly, but this UT call compaction directly, did not walk through the wait logic.

Reply:
I think call the wait may be a better way to handle the exception.

Reply:
this is the wait definitely, the _wait_for_version is a private function, this get_applied_rowsets is a wrapper for it, it will call _wait_for_version.